bench_runner/result.py

```diff
@@ -523,6 +523,21 @@ def github_action_url(self) -> Optional[str]:
     def is_tier2(self) -> bool:
         return "PYTHON_UOPS" in self.flags
 
+    @property
+    def is_jit(self) -> bool:
+        return "JIT" in self.flags
+
+    @property
+    def hash_and_flags(self) -> str:
+        # A representation for the user that combines the commit hash and other flags
+        return " ".join(
+            [
+                self.cpython_hash,
+                *(self.is_tier2 and ["2️⃣"] or []),
+                *(self.is_jit and ["*️⃣"] or []),
+            ]
+        )
```

Comment:
I find this a bit easier to follow:

```suggestion
        parts = [self.cpython_hash]
        if self.is_tier2:
            parts.append("2️⃣")
        if self.is_jit:
            parts.append("*️⃣")
        return " ".join(parts)
```